addons/l10n_din5008/models/base_document_layout.py

```diff
@@ -1,10 +1,34 @@
-from odoo import models, fields, _
+from markupsafe import Markup
+import re
+
+from odoo import api, models, fields, _
 from odoo.tools import format_date
 
 
 class BaseDocumentLayout(models.TransientModel):
     _inherit = 'base.document.layout'
 
+    @api.model
+    def _default_report_footer(self):
+        # OVERRIDE web/models/base_document_layout
+        if self.env.company.external_report_layout_id == self.env.ref('l10n_din5008.external_layout_din5008'):
+            company = self.env.company
+            # Company VAT should not be present in this footer, as it is displayed elsewhere in the DIN5008 layout
+            footer_fields = [field for field in [company.phone, company.email, company.website] if isinstance(field, str) and len(field) > 0]
+            return Markup('<br>').join(footer_fields)
+        return super()._default_report_footer()
```

Comment:
@odoo/rd-security security warning about these lines (and the use of `Markup`). Should be OK since this code is (mostly) the same as the method it overrides.

Reply:
> Should be OK since this code is (mostly) the same as the method it overrides.

That is improper reasoning, the method it overrides could be completely wrong ;)

This is ok because the `Markup` is only applied to a static string, which is safe by definition :smile:

Reply:
> This is ok because the `Markup` is only applied to a static string

Isn't this easy to check with a linter rule to avoid useless pings to security team?

Reply:
We plan to use [semgrep](https://semgrep.dev) to avoid these.